src/MOI_wrapper.jl

```diff
@@ -196,25 +196,58 @@ const _SUPPORTED_MODEL_ATTRIBUTES = Union{
     # MOI.RawStatusString,  # TODO
     MOI.ResultCount,
     # MOI.Silent, # TODO
-    # MOI.TerminationStatus,
+    MOI.TerminationStatus,
     # MOI.PrimalStatus,
     # MOI.DualStatus
 }
-
+@enum HiGHS_ModelStatus begin
```

Comment:
I would call this `HighsModelStatus` to match the C code: https://github.com/ERGO-Code/HiGHS/blob/bf336493c1353e6a60aca47eff353c76599a077b/src/lp_data/HConst.h#L114-L135

There's a question on whether we want to use https://github.com/JuliaInterop/CEnum.jl, but this is probably okay for now.